components/LibraryCard/LibraryCard.module.scss

```diff
@@ -26,11 +36,17 @@
   a {
     text-decoration: none;
   }
+
+  @media only screen and (max-width: $mobile-width) {
+    background-color: hsl(var(--white)/12%);
+    box-shadow: var(--box-shadow);
+  }
 }
 
 .content {
   .image {
     position: relative;
+    transition: all .2s linear;
```

Comment:
Still need to remove this transition.

Reply:
Transition smoothes the behavior of the icon when hovering over the card.

Reply:
What I'm asking is to remove all transitions from those cards. Keep the style as you have proposed before hover, but make the buttons the only clickable areas. 😄